src/style.css

```diff
@@ -1,3 +1,69 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  --active-color: #00ACDC;
```

Comment:
better to move all vars in `::root`

Reply:
Thanks for the feedback. Should I create a new commit on github or just fix the code myself?